build/build-viewer.js

```diff
@@ -14,9 +14,6 @@ async function buildReportGenerator() {
   const bundle = await rollup.rollup({
     input: 'report/generator/report-generator.js',
     plugins: [
-      rollupPlugins.shim({
-        [`${LH_ROOT}/report/generator/flow-report-assets.js`]: 'export default {}',
-      }),
```

Comment:
this moves the bundle from 536K to 1.15M :(

Maybe we should look into making this whole report generation feature async so we can lazily load this on Save as HTML :/ 

Don't intend to have this block the PR

Reply:
Disabled HTML saving for flows in viewer for this PR